### PR TITLE
Introduces base for AdEx staking page

### PR DESCRIPTION
**Pull request by kelsos**
Based on #1880

### CODE REVIEW

**Review by LefterisJP — COMMENTED** [review]
Hey @kelsos 


Thanks a lot. So from a first look:

## Not load Eth2 by default
Just like with DeFi menu entry, now that we are adding more options on this we should not load anything (it loads ETH2) by default.

Instead we should allow the user to choose just like we do with the defi meny.


## Bond Id

I don't think the bond Id column is required. If we need space it can go away.

## Show unclaimed balance

Here you could show the unclaimed balance. This is something that @vnavascues should put in the api.



So split it into staked balance, and unclaimed rewards like they do in their UI.

## Show total profit so far

Also at the same place show the total profit from ADX staking so far. This should also be added to the api by @vnavascues

**Review by LefterisJP — COMMENTED** [review]
I added 2 backend changes since the balances were not showing correctly.

So looks good to me @kelsos apart from one very needed change in the events table.
The timestamp is missing.



Remove identity address and use the timestamp instead. It's really confusing without it.

**Comment by kelsos** [comment]
@LefterisJP ok so we can merge this PR then and I only need to change the components.

File: rotkehlchen/chain/manager.py [hunk 1]
```diff
@@ -924,7 +924,9 @@ def _add_protocol_balances(self) -> None:
             for address, pool_balances in adex_balances.items():
                 for pool_balance in pool_balances:
                     eth_balances[address].assets[A_ADX] += pool_balance.adx_balance
+                    self.totals.assets[A_ADX] += pool_balance.adx_balance
```
Comment (LefterisJP) [inline rotkehlchen/chain/manager.py:927]: @vnavascues this is something you forgot to add when taking adex balances into account. Keep that in mind for next
time when including balances from a protocol.

We got per account balances and then we got total balances and adding should happen for both.

**Review by LefterisJP — APPROVED** [review]
Required change is only in premium components.

**Pull request merged**